docs/contributing/testing-guidelines/index.md

```diff
@@ -1,5 +1,23 @@
 # Testing guidelines
 
-!!! warning
+## Unit tests
 
-    Under Construction
+Unit Testing is a software testing method by which individual units of source code are tested to determine whether they are fit for use.
+
+The tool used for unit testing in Autoware is `gtest`.
```

Comment:
By writing this here, what do we expect users to learn?
I guess we can drop it from this page.

```suggestion
```